components/prism-javadoc.js

```diff
@@ -0,0 +1,56 @@
+(function (Prism) {
+
+	var javaDocLike = Prism.languages.javadoclike;
```

Comment:
I remember we're doing this for minification reasons, but if we're only using it once, do we actually get those benefits here?

Reply:
You're right! I will remove it.